apps/avifdec.c

```diff
@@ -225,6 +229,10 @@ int main(int argc, char * argv[])
                 returnCode = 1;
             }
         } else if (outputFormat == AVIF_APP_FILE_FORMAT_JPEG) {
+            // Bypass alpha multiply step during conversion
```

Comment:
Hi Tong Yuan,

The help message says raw color is always true for y4m. If so, shouldn't we always set avif->alphaPremultiplied to AVIF_TRUE between lines 227 and 228?

Reply:
`avif->alphaPremultiplied` only affects the conversion between RGB and YUV, and since y4m stores YUV data, this value is unrelated.